common/changes/@bentley/eslint-plugin/eslint-semi_2020-11-10-14-15.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/eslint-plugin",
+      "comment": "",
```

Comment:
@paulius-valiunas can you put a comment in here about the change in defaults for imodeljs-recommended?  Consumers of the plugin are going to hit this as well.

Reply:
is this better?